client/src/components/TextInputComponent/TextInput.css

```diff
@@ -0,0 +1,17 @@
+@font-face {
+  font-family: Raleway;
+  src: url("https://fonts.googleapis.com/css?family=Raleway&display=swap");
+}
+
+.textInputClass {
```

Comment:
CSS class names should be all lower case and separated by dashes (i.e. `text-input-class`, rather than `textInputClass`). Also it's redundant to put `class` in the name of a CSS class!

Reply:
Completely agree there